tests/test_clone.py

```diff
@@ -0,0 +1,249 @@
+from lib.elements import Clone, EmbroideryElement
+from lib.svg.tags import INKSTITCH_ATTRIBS
+from inkex import SvgDocumentElement, Rectangle, Circle, Group, Use, Transform, TextElement
+from inkex.tester import TestCase
+from inkex.tester.svg import svg
+
+from math import sqrt
+
+
+def element_fill_angle(element: EmbroideryElement) -> float | None:
```

Comment:
I'm not sure if we really need to comply within the tests with our build rules, but I rather would. On multiple requests we have to stay with python 3.8 to still support Windows 7. We may have to drop the support anyway soon, but could you update this to Optional[float] instead (to support python 3.8).

Also it'd be nice if you could add a init file to the test folder. Otherwise I it refused to run pytest on the folder which would be a pity sine we have your nice test file in there now.

Reply:
I thought I added that, but whoops, I see I didn't. Added it, along with making the change.